src/main/java/org/folio/rest/impl/DataExportImpl.java

```diff
@@ -64,13 +64,18 @@ public void postDataExportExport(ExportRequest entity, Map<String, String> okapi
         jobProfileService.getById(entity.getJobProfileId(), tenantId)
           .onSuccess(jobProfile ->
             mappingProfileService.getById(jobProfile.getMappingProfileId(), tenantId)
-              .onSuccess(mappingProfile -> {
-                inputDataManager.init(JsonObject.mapFrom(entity), JsonObject.mapFrom(requestFileDefinition), JsonObject.mapFrom(mappingProfile), okapiHeaders);
-                succeededFuture()
-                  .map(PostDataExportExportResponse.respond204())
-                  .map(Response.class::cast)
-                  .onComplete(asyncResultHandler);
-              })
+              .onSuccess(mappingProfile ->
+                jobExecutionService.getById(requestFileDefinition.getJobExecutionId(), tenantId)
```

Comment:
we need to refactor this to a helper class or something next time we push anything, getting hard to read. 
we can leave it as is for now